docs/src/main/asciidoc/tests-with-coverage.adoc

```diff
@@ -191,6 +191,8 @@ specifying a `data-file` or `report-location` will assume the path as is. Here i
 </plugin
 ----
 
+WARNING: if you need to configure the `argLine` property on the surefire plugin (e.g. setting memory parameters), you need to use https://maven.apache.org/surefire/maven-surefire-plugin/faq.html#late-property-evaluation[maven late property evaluation^], otherwise the jacoco agent will not be correctly added, and regular JUnit tests and Quarkus `ComponentTest` will not get covered. Example: `<argLine>@{argLine} -your -extra -arguments</argLine>`
```

Comment:
```suggestion
WARNING: If you need to configure the `argLine` property of the Surefire plugin (e.g. for setting memory parameters), you need to use https://maven.apache.org/surefire/maven-surefire-plugin/faq.html#late-property-evaluation[Maven late property evaluation^], otherwise the Jacoco agent will not be correctly added, and regular JUnit tests and Quarkus `ComponentTest` will not get covered. Example: `<argLine>@{argLine} -your -extra -arguments</argLine>`.
```